include/openssl/core_numbers.h

```diff
@@ -281,6 +281,36 @@ OSSL_CORE_MAKE_FUNC(int, OP_mac_get_ctx_params,
 OSSL_CORE_MAKE_FUNC(int, OP_mac_set_ctx_params,
                     (void *mctx, const OSSL_PARAM params[]))
 
+/* KDFs and PRFs */
+# define OSSL_OP_KDF                                 4
```

Comment:
It seems that we're grouping all the `_OP_` macros in one place, up around line 155...

Reply:
Yep, done.